README.md

```diff
@@ -280,6 +280,34 @@ from prometheus_client import start_wsgi_server
 start_wsgi_server(8000)
 ```
 
+#### Flask
+
+To use prometheus with [flask](http://flask.pocoo.org/) we need to serve metrics through a promethues wsgi application. This can be achieved using [flask's application dispatching](http://flask.pocoo.org/docs/latest/patterns/appdispatch/). Below is a working example.
+
+```python
+from flask import Flask
+from werkzeug.wsgi import DispatcherMiddleware
+from prometheus_client import make_wsgi_app
+
+# Create my app
+app = Flask(__name__)
+
+# Add prometheus wsgi middleware to route /metrics requests
+app_dispatch = DispatcherMiddleware(app, {
+    '/metrics': make_wsgi_app()
+})
+```
+
+To test this example, save the above snippet in a `myapp.py` file and run the command below
```

Comment:
We mention filenames before the relevant example.

Reply:
Done